schema-engine/core/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub mod commands;
 mod api;
 mod core_error;
 mod rpc;
-mod state;
+pub mod state;
```

Comment:
I will undo this. At some point ran the migrations by importing schema core. No longer needed.